docs/contributions.md

```diff
@@ -1,157 +1,200 @@
+| Security flaw: permissions escalation                                                                                  | expose user to malware                                                                   | 5        |
+| Code not addressing user needs                                                                                         | no product market fit, loss of adoption                                                  | 5        |
+| Communication failures within team                                                                                     | Conflicting or duplicating work, frustration, morale damage                              | 4        |
+| Communication failures with community                                                                                  | we lose product market fit, we do not have good community retention, reputational damage | 3        |
+| Communication failures with leadership                                                                                 | product does not meet business goals                                                     | 3        |
+| License compliance failures, including in supply chain (e.g. exposing privileged api responses for free via  a plugin) | 4                                                                                        |
+| Bugs affecting releases                                                                                                | users stuck on old versions                                                              | 2        |
+| Strategy failures                                                                                                      | no product market fit                                                                    | 2        |
+
+
+The mapping of severity to label is as follows:
+
+| Severity | Label |
+| -------- | ----- |
+| 1        | L     |
+| 2        | M     |
+| 3        | M     |
+| 4        | H     |
+| 5        | H     |
+
+During the bug triage we will also discuss a remediation plan for the bug. This will be communicated in the comments on the bug report. For high urgency bugs, the fix will be implemented as soon as possible, maybe reorganizing our current work to accommodate it. For medium priority bugs, we will schedule the fix in the next available sprint. Low priority bugs will be backlogged and addressed when there is developer time available. Low priority bugs will also be tagged `help-wanted` so that they can be addressed by community members.
+
+Not every bug will be fixed. We may decide *not* to fix a bug in cases such as:
+
+- fixing the bug has some detrimental side effect elsewhere in the product
+- the bug has a fix coming soon as part of another upgrade
+- the bug is only problematic for a single specific use casea nd fixing it would break features relied upon by other users
```

Comment:
@jmcook1186 casea nd fixing --> case and fixing

Reply:
```suggestion
- the bug is only problematic for a single specific use case and fixing it would break features relied upon by other users
```